native/cmake/predefine.cmake

```diff
@@ -122,6 +129,10 @@ else()
     message(STATUS "Ignore NO_WERROR")
 endif()
 
+if("${OHOS_ARCH}" STREQUAL "armeabi-v7a")
+    message("OHOS_ARCH1 : ${OHOS_ARCH}")
+    set(CMAKE_CXX_FLAGS "-march=armv7a -fvisibility=hidden -fvisibility-inlines-hidden ${CMAKE_CXX_FLAGS}")
```

Comment:
This flags can be set for all architecturs.  refer to changes in https://github.com/cocos/cocos-engine/pull/11792/files